.github/workflows/deploy-docs.yml

```diff
@@ -23,6 +23,8 @@ jobs:
          conda create --name IOOS --file .binder/conda-linux-64.lock
          source activate IOOS
          conda install doctr
+      ## Adding install jupyter{book} command 
```

Comment:
This is REALLY a minor issue. But I kind of like the comment `#` to be aligned with the code. We can fix that later though.